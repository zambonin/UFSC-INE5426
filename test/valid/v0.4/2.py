exec(open('../../../src/scope_manager.py', 'r').read())

s_context()
if True:
    s_context()
    if (2.3 <= float(4)):
        s_context()
        if ((2 + 3) >= 5):
            f = 2.0
        r_context()
    else:
        f = 1.0
    r_context()
r_context()
