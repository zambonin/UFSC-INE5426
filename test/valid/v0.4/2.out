float var: f
if: true
then:
  if: <= 2.3 [float] 4
  then:
    if: >= + 2 3 5
    then:
      = f 2.0
  else:
    = f 1.0
